src/install/install.zig

```diff
@@ -1079,29 +1080,28 @@ pub fn NewPackageInstall(comptime kind: PkgInstallKind) type {
             bun.debugAssert(!this.patch.isNull());
 
             // hash from the .patch file, to be checked against bun tag
-            const patchfile_contents_hash = this.patch.patch_contents_hash;
-            var buf: BuntagHashBuf = undefined;
-            const bunhashtag = buntaghashbuf_make(&buf, patchfile_contents_hash);
+            var patch_buf: BunTagFile.Buffer = undefined;
+            const tag_filename = BunTagFile.path(&patch_buf, this.patch.patch_contents_hash);
 
-            const patch_tag_path = bun.path.joinZ(&[_][]const u8{
-                this.destination_dir_subpath,
-                bunhashtag,
-            }, .posix);
+            if (comptime bun.Environment.isPosix) {
```

Comment:
does `existsAt` not support subpaths on windows? if so we should add a comment about this

Reply:
and a debug assertion